docs/README-freebsd.md

```diff
@@ -4,27 +4,44 @@
 
 ### Prerequisites
 
-Make sure you've UTF-8 defined for charset and lang in your [~/.login_conf](https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/using-localization.html),
-otherwise almost every module will fail to install. Python 3 is very sensitive to that.
+Make sure you've UTF-8 defined for charset and lang in your [~/.login_conf](https://www.freebsd.org/doc/en_US.ISO8859-1/books/handbook/using-localization.html), 
+otherwise almost every Python 3 module will fail to install.
 
+`~/.login_conf`:
 ```
-sudo pkg install python3 py36-virtualenv leveldb libxml2 libxslt pkgconf secp256k1 wget git
-sudo wget https://raw.githubusercontent.com/bitcoin/bitcoin/master/src/secp256k1/include/secp256k1_recovery.h -O /usr/local/include/secp256k1_recovery.h
```

Comment:
Can you explain a little more why this was here before, and not necessary anymore?

Reply:
@carver, for pkg install - all packages are still there. For secp256k - I've submitted a fix into secp256k port (https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=233456). Basically now secp256k1_recovery.h file will be copied during secp256k package installation as it should be so we don't need to download it manually. Thanks.

Reply:
Hm, weird... I don't see couincurve as a dependency anywhere in this project. Probably from the very beginning everything related to coincurve\secp256k1 were related to py-evm project only.

So for web3.py secp256k1 installation is not needed.

I've performed all steps on a freshly installed OS image and they passed with no errors (I've double-checked it now again).

Reply:
For anything involving local signing (`w3.eth.account`), coincurve will make things run notably faster, IIRC. So it's probably still worth having.